format/IPC.md

```diff
@@ -189,6 +197,10 @@ in the schema, so that dictionaries can even be used for multiple fields. See
 the [Physical Layout][4] document for more about the semantics of
 dictionary-encoded data.
 
+The dictionary `isDelta` flag allows dictionary batches to be modified mid-stream.
+A dictionary batch with `isDelta` set indicates that its vector should be
+concatenated with those of any previous batches with the same `id`.
```

Comment:
For the avoidance of confusion, we might want to include a small pseudocode example of a stream with a delta dictionary batch

Reply:
Would something like the example in my initial email work?